test/evergreen.yml

```diff
@@ -1952,6 +1973,20 @@ tasks:
           format_test_script_args: -b "SEGFAULT_SIGNALS=all catchsegv ./t" -S
           times: 16
 
+  - name: checkpoint-stress-test
+    commands:
+      - command: timeout.update
+        params:
+          # set 24 hour timeout for the task and command
+          exec_timeout_secs: 86400
+          timeout_secs: 86400
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - func: "checkpoint stress test"
+        vars:
+          times: 2
```

Comment:
Could a comment be added to explain the rationale of the value setting here?

Reply:
Fixed

Reply:
I actually meant to explain why 2 was selected e.g. the measured runtime of 2 runs is close to 24hrs to simulate continuously running job in Jenkins